src/csharpTranspiler.ts

```diff
@@ -885,6 +885,10 @@ export class CSharpTranspiler extends BaseTranspiler {
         return `((string)${name}).Split(new [] {((string)${parsedArg})}, StringSplitOptions.None).ToList<object>()`;
     }
 
+    printConcatCall(node, identation, name = undefined, parsedArg = undefined) {
+        return `${name}.Concat(${parsedArg}).ToList()`;
```

Comment:
@ttodua which concat do you want to use here? out of the box this is not going to work

Reply:
in that case we will need to cast array1 first maybe to IList<object> but that won't work if the type is different.

Reply:
if types are different then it will not work ofc, but atm we heavily use `.concat` in ccxt, and what should we do with astt, to support it?

Reply:
> In this case I think it's easier to add a helperMethod, to handle the different types. I will check it

good, tnx